warp/cairo-src/evm/calls.cairo

```diff
@@ -1,16 +1,31 @@
-from starkware.cairo.common.uint256 import Uint256
+from starkware.cairo.common.dict_access import DictAccess
 from starkware.cairo.common.math import split_felt
+from starkware.cairo.common.uint256 import Uint256
 from starkware.starknet.common.syscalls import get_caller_address
-from evm.exec_env import ExecutionEnvironment
+
+from evm.array import array_copy_to_memory
 from evm.array import array_load
+from evm.exec_env import ExecutionEnvironment
+from evm.utils import update_msize
 
-func get_caller_data_uint256{syscall_ptr : felt*, range_check_ptr}() -> (caller_data: Uint256):
-  alloc_locals
-  let (local caller_address) = get_caller_address()
-  let (local high, local low) = split_felt(caller_address)
-  let caller_data = Uint256(low=low, high=high)
+func get_caller_data_uint256{syscall_ptr : felt*, range_check_ptr}() -> (caller_data : Uint256):
+    alloc_locals
+    let (local caller_address) = get_caller_address()
+    let (local high, local low) = split_felt(caller_address)
+    let caller_data = Uint256(low=low, high=high)
+
+    return (caller_data=caller_data)
+end
 
-  return (caller_data=caller_data)
+func calldatacopy_{
```

Comment:
The naming with underscore for `mstore` and `mload` was introduced not to break compatibility with the EVM transpiler. Fortunately, it's not a concern here. Anyway, I'm just carping, it doesn't matter